packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -35,7 +35,8 @@
     "cross-env": "^5.0.5",
     "mocha": "3.2.0",
     "sinon": "^2.3.6",
-    "vscode": "1.1.17"
+    "vscode": "1.1.17",
+    "nyc": "^11.0.2"
```

Comment:
Can we set the `nyc` version without using the carrot ? Also, is this the same `nyc` version used by the sfdx cli ?

Reply:
We can use the same version as sfdx cli- I'm going to try configuring ours using `ts-node` as well so this dependency will be removed. In case it doesn't work, I'll update this version number.